src/Client.js

```diff
@@ -836,6 +836,7 @@ class Client extends EventEmitter {
      * @property {string} [stickerName=undefined] - Sets the name of the sticker, (if sendMediaAsSticker is true).
      * @property {string[]} [stickerCategories=undefined] - Sets the categories of the sticker, (if sendMediaAsSticker is true). Provide emoji char array, can be null.
      * @property {MessageMedia} [media] - Media to be sent
+     * @property {any} [extra] - Extra options
```

Comment:
What is that?

Reply:
@shirser121 
> What is that?

This is [extra options](https://github.com/pedroslopez/whatsapp-web.js/pull/600) to pass